docs/userguide/scheduling/cluster-resources.md

```diff
@@ -351,24 +352,24 @@ status:
     grade: 6   
 ```
 
-Suppose the Pod's resource request is for a 3-cores CPU and 20Gi of memory. All nodes that meet Grade 2 and above fulfill this request. Considering the number of nodes available in each cluster, the scheduler prefers to schedule the Pod to member3.
+Suppose the Pod's resource request is for a 3-core CPU and 20Gi of memory. All nodes that are classified as grade 3 and above fulfill this request, since we need grade's resource `min` value to be at least as big as the requested value. For example, nodes in grade 2 with less than 3C and 20Gi don't fulfill our requirements, so we eliminate the entire grade due to that. Scheduler will then choose the cluster which can fit the most replicas on its nodes, which in our case is cluster 3, as computed below:
```

Comment:
> Scheduler will then choose the cluster which can fit the most replicas on its nodes, which in our case is cluster 3, as computed below

Maybe we don't need to say the intention of which clusters it will choose, because it depends on the scheduling policy, we just say the result of available replicas each cluster can hold.

Reply:
Ok, I can change that, scheduling result was mentioned in the original english text, so I included it too.

Reply:
Something like:
```
The Cluster Resource Model classifies nodes within a cluster based on their available resources. This classification helps the scheduler determine the number of replicas that can fit into each cluster. By categorizing nodes according to their available resources, the Cluster Resource Model enables the scheduler to make decisions about where to place application replicas based on different scheduling strategies.
```